lib/spack/docs/requirements.txt

```
@@ -8,3 +8,4 @@ python-levenshtein
 # Restrict to docutils <0.17 to workaround a list rendering issue in sphinx.
 # https://stackoverflow.com/questions/67542699
 docutils <0.17
+pygments <=2.12.0
```

```suggestion
pygments <2.13
```
I would use this in case 2.12.1 comes out before we fix the original issue.